### DISCUSSION TITLE
Can't Connect Plugin to Server

### DISCUSSION THREAD

**Discussion by damondahl** [apohl79/audiogridder#1239] [General]
I am using my Windows 10 Machine (CPU: AMD, GPU: NVIDIA) as an audiogridder Server.

I want to use my MacBook as the main host for the audiogridder Plugin. 

I am having issues trying to get my MacBook to recognize the Server on the Windows Machine. They are BOTH connected via ethernet on the same Router.

I have tried entering the IP Address and the Server name. -Nothing has worked yet so far.

What am I doing wrong?

*yes. I do have plugins installed on the Windows Server*

**Comment by apohl79** [comment]
Sounds like a windows firewall issue. You can temporarily disable it to confirm.

**Reply by paki59** [reply]
hI.  eXAMPLE: LAPtop you install AG plug in and in option internet ip4 set ip 192.168.1.1  in server install Ag SERVER  SET INTERNET  ip4  with ip 192.168.1.2.  Opens server plug ing that set all vst and plug in. After open yr Daw (Cubase or Logic or other daw)in laptop...open in vst Audio gridder instrument track, set ip 192.168.1.2 (server) and you will see all vst and plug in...
or at least most of them. Bye

**Reply by damondahl** [reply]
(as you already knew) YOU WERE RIGHT. But now another problem. How can I get this to startup and run consistently WITHOUT having to disable my firewall?

**Reply by paki59** [reply]
I use daw server and master without firewall and antivirus because they slow down the system and I'm not connected to the internet, so suspend firewall and antivirus...

**Comment by oldtos** [comment]
I use Tinywall .. it has an option to allow/unblock LAN traffic... .. your fire wall may also have options to allow certain traffic..
